src/v/config/tests/bounded_property_test.cc

```diff
@@ -12,20 +12,25 @@
 
```

Comment:
nit: should probably go after the next commit?

Reply:
right. i wanted to revise this anyway